client/plugins/fcc-source-challenges/create-challenge-nodes.js

```diff
@@ -33,7 +33,8 @@ function createChallengeNode(challenge, reporter) {
           internal,
           sourceInstanceName: 'challenge'
         },
-        challenge
+        challenge,
+        { id: challenge.id + internal.type }
```

Comment:
It was my understanding this needed to be changed? Is this an old rebase?

Reply:
I've updated it now to reflect the `upstream`. 

Should I pull all the changes from the upstream and merge it with the fork? The fork seems to be quite outdated.

Reply:
> Should I pull all the changes from the upstream and merge it with the fork?

I cannot see why not to do this.

Reply:
Done